.travis.yml

```diff
@@ -30,6 +30,10 @@ script: sbt -jvm-opts .jvmopts-travis -Dakka.build.scalaVersion=$TRAVIS_SCALA_VE
 
 jobs:
   include:
+    - stage: test
+      name: compile with java 14
+      script: jabba install "openjdk@~1.14.0" && jabba use openjdk@~1.14.0 && sbt -jvm-opts .jvmopts-travis -Dakka.build.scalaVersion=$TRAVIS_SCALA_VERSION "test:compile"
```

Comment:
instead of overwriting the script, could we set `TRAVIS_JDK=11` in this job? or are there problems with some of the other parts?